### ISSUE TITLE
Add Review option to banner

### ISSUE THREAD

**Issue by javimuu** [ain/smartbanner.js#50]
Hello. Can we add review option (star and reviews count) like image bellow:

![smartbanner_2x](https://cloud.githubusercontent.com/assets/9436541/26386249/7996a3e8-4080-11e7-98c4-dd32ef24229b.png)

Thank you.

**Comment by ain** [comment]
Thanks for the question @javimuu.

The reason why there's no ratings integration is simple: we don't have the data. If there was App Store's JavaScript API from which we could fetch data, it'd be doable, but publishing the stars based on manual data entry would be unethical.

On these considerations I'm closing the ticket, but feel free to continue the discussion in case you have additional concerns.

**Comment by sebastianhelbig** [comment]
Just for consideration:
There are tools (i.e. appbot) which can scrap the reviews of an app. So it would be possible to fetch the current star rating serverside (outside of smartbanner.js) for displaying it with smartbanner.js.

Btw. Thank you for this nice library.

**Comment by ain** [comment]
Thanks for your input @sebastianhelbig! On these grounds we can reopen the ticket.

**Comment by jnsntm** [comment]
Any updates on this? Would be super nice to have that function.

**Comment by ain** [comment]
@jnsntm planned for `v2.0.0` as per the linked milestone. Work is pending.

**Comment by KasperAndersson** [comment]
Hi @ain , 

Is this project left?

Thanks in advance.

**Comment by ain** [comment]
@KasperAndersson no, there has been no capacity to develop this. You're welcome to send a PR!
